process/process.go

```diff
@@ -165,9 +171,8 @@ func parseMappings(mapsFile io.Reader) ([]Mapping, uint32, error) {
 				path = VdsoPathName
 				device = 0
 				inode = vdsoInode
-			} else if path != "" {
-				// Ignore [vsyscall] and similar executable kernel
-				// pages we don't care about
+			} else {
```

Comment:
No semantic change, I just inlined the logic from `GetMappings` here as this is the more appropriate place.